src/status.rs

```diff
@@ -453,14 +455,16 @@ impl StatusBuilder {
     }
 
     pub fn build(self) -> Result<Status, String> {
-        Ok(Status::new(
-            self.space,
-            self.logo.ok_or("logo missing")?,
-            self.url.ok_or("url missing")?,
-            self.location.ok_or("location missing")?,
-            self.contact.ok_or("contact missing")?,
-            self.issue_report_channels,
-            ))
+        Ok(Status {
+            api: "0.13".into(), // TODO: Deduplicate
```

Comment:
The validator doesn't use this crate at all. Also we *can't* de-serialize to any other SpaceAPI version, since it would break anyway. So we could just throw an "unsupported SpaceAPI" version error.